File: Dtos/Commands/EditCourseCommand.cs
using System;

namespace OnlineCourse.Api.Dtos.Commands
{
    /// <summary>
    /// edit command
    /// </summary>
    public class EditCourseCommand
    {
        /// <summary>
        /// Gets or sets the name.
        /// </summary>
        /// <value>
        /// The name.
        /// </value>
        public string Name { get; set; }

		/// <summary>
		/// gets or sets MaxParticipants
		/// </summary>
		public int MaxParticipants { get; set; }

		/// <summary>
		/// gets or sets Teacher
		/// </summary>
		public string Teacher { get; set; }

		/// <summary>
		/// Gets or sets the Course unique identifier.
		/// </summary>
		/// <value>
		/// The Course unique identifier.
		/// </value>
		public Guid CourseGuid { get; set; }
    }
}

File: UI/src/app/models/editCourseCommand.ts
import {Guid} from "../utils/CreateGuid";

export class EditCourseCommand {
  courseGuid: string;
  name: string;
  teacher: string;
  maxParticipants: number;
  constructor( courseGuid: string, name: string, maxParticipants: number, teacher: string) {
    this.courseGuid = courseGuid;
    this.name = name;
    this.maxParticipants = maxParticipants;
    this.teacher = teacher;
  }
}
